pkgs/servers/web-apps/searx/default.nix

```diff
@@ -4,20 +4,26 @@ with python3Packages;
 
 toPythonModule (buildPythonApplication rec {
   pname = "searx";
-  version = "0.17.0";
+  version = "0.18.0";
 
   # Can not use PyPI because certain test files are missing.
   src = fetchFromGitHub {
-    owner = "asciimoo";
+    owner = "searx";
     repo = "searx";
     rev = "v${version}";
-    sha256 = "0pznz3wsaikl8khmzqvj05kzh5y07hjw8gqhy6x0lz1b00cn5af4";
+    sha256 = "0idxspvckvsd02v42h4z4wqrfkn1l8n59i91f7pc837cxya8p6hn";
   };
 
+  patches = [
+    ./test_json_serial.patch
+  ];
+
   postPatch = ''
     sed -i 's/==.*$//' requirements.txt
   '';
 
+  SEARX_DEBUG = "true";
```

Comment:
Can you export this in the pre or post phase where it is required?

Reply:
I'll verify it builds that way and commit.